composer.json

```
@@ -1,5 +1,5 @@
 {
-    "name": "Photo Sphere Viewer",
+    "name": "files_photospheres",
```

Note that the package name is not well-formed to match the specification of the upcoming Composer 2.0 version:
```
Deprecation warning: Your package name files_photospheres is invalid, it should have a vendor name, a forward slash, and a package name. The vendor and package name can be words separated by -, . or _. The complete name should match "^[a-z0-9]([_.-]?[a-z0-9]+)*/[a-z0-9](([_.]?|-{0,2})[a-z0-9]+)*$". Make sure you fix this as Composer 2.0 will error.
```

One have to set a vendor name and a forward slash in front of the `files_photospheres` name, eg.:
```suggestion
    "name": "nextcloud/files_photospheres",
```